tasks/deploy/config/sepolia/dai-14day.ts

```diff
@@ -1,121 +1,119 @@
 import { parseEther, toFunctionSelector } from "viem";
-import { HyperdriveInstanceDeployConfigInput } from "../../lib";
-const CONTRIBUTION = "10000";
-export const SEPOLIA_DAI_14DAY: HyperdriveInstanceDeployConfigInput = {
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+
+const CONTRIBUTION = parseEther("10000");
+
+export const SEPOLIA_DAI_14DAY: HyperdriveInstanceConfig<"ERC4626"> = {
     name: "DAI_14_DAY",
-    contract: "ERC4626Hyperdrive",
-    coordinatorName: "ERC4626_COORDINATOR",
-    deploymentId: "0x666",
-    salt: "0x69420",
+    prefix: "ERC4626",
+    coordinatorAddress: async (hre) =>
+        hre.hyperdriveDeploy.deployments.byName("ERC4626_COORDINATOR").address,
+    deploymentId: toBytes32("DAI_14_DAY"),
+    salt: toBytes32("0x69420"),
+    extraData: "0x",
     contribution: CONTRIBUTION,
-    fixedAPR: "0.05",
-    timestretchAPR: "0.05",
+    fixedAPR: parseEther("0.10"),
+    timestretchAPR: parseEther("0.10"),
+    targetCount: 4,
     options: {
-        // destination: "0xsomeone", defaults to deployer
+        extraData: "0x",
         asBase: true,
-        // extraData: "0x",
+        destination: "0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8",
```

Comment:
It's fine to leave this as is for now, but it would be good for this to be the `msg.sender` in the future

Reply:
Agreed, in the same vein as [this](https://github.com/delvtech/hyperdrive/pull/1019#discussion_r1604144477) but easier, can tackle at a later point in time